### PR TITLE
Add bigswitch big mon inline chain module

### PR DESCRIPTION
**Pull request by tedelhourani**
##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

Utils for big switch network modules

##### ANSIBLE VERSION

```
ansible 2.1.2.0
```

##### SUMMARY

Utils based on urls to be consumed by the big switch modules

```

```

### CODE REVIEW

**Comment by tedelhourani** [comment]
This PR has one module with the utils stub it depends on.

File: lib/ansible/modules/network/bigswitch/bigmon_chain.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    required: true
+  state:
+    description:
+     - Whether the service chain should be present or absent.
+    default: present
+    choices: ['present', 'absent']
+  access_token:
+    description:
+     - Bigmon access token.
+
```
Comment (privateip) [inline lib/ansible/modules/network/bigswitch/bigmon_chain.py:58]: missing required:

File: lib/ansible/modules/network/bigswitch/bigmon_chain.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        if response.status_code == 204:
+            module.exit_json(changed=True)
+        else:
+            module.fail_json(msg="error deleting chain '{}': {}".format(name, response.json['description']))
+
+def main():
+    module = AnsibleModule(
+        argument_spec=dict(
+            name=dict(type='str', required=True),
+            controller=dict(type='str', required=True),
```
Comment (privateip) [inline lib/ansible/modules/network/bigswitch/bigmon_chain.py:145]: missing `controller` in DOCUMENTATION

File: lib/ansible/modules/network/bigswitch/bigmon_chain.py [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+            state=dict(choices=['present', 'absent'], default='present'),
+            validate_certs=dict(type='bool', default='False'),
```
Comment (privateip) [inline lib/ansible/modules/network/bigswitch/bigmon_chain.py:147]: missing `validate_certs` in DOCUMENTATION

**Review by privateip — CHANGES_REQUESTED** [review]
* bigswitch_utils.py is missing the BSD license header
* bigmon_chain is missing metadata
* bigmon_chain is missing author

**Comment by privateip** [comment]
needs_revision

**Comment by tedelhourani** [comment]
Made the revisions, but Shippable is timing out

**Review by privateip — APPROVED** [review]

**Comment by privateip** [comment]
Thanks... merged into devel

**Pull request merged**